app/components/ui/Navigation.module.scss

```diff
@@ -172,3 +178,29 @@
     font-style: normal;
   }
 }
+
+.buttonLink {
+  appearance: button;
+  text-decoration: none;
+  color: white;
+  background-color: $color-brand;
+  border-radius: 3px;
+  font-size: 16px;
+  font-weight: 600;
+  height: 60px;
+  width: 200px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  &:focus,
+  &:hover {
+    color: white;
```

Comment:
~~I'm guessing this is one of the pieces that you said was copied and pasted from the Search Button, but this line doesn't have any effect since the non-pseudoclass styles already set it to white~~ Nevermind, I thought this wasn't necessary, but I forgot we have some global styles that we keep on having to override. I'm still going to post this comment, even though there's no action to take right now, but I just wanted something to link to in https://github.com/ShelterTechSF/askdarcel-web/issues/1177, which is a ticket I created somewhat recently to clean up our global CSS styles.